src/screens/Analytics/GlobalSearch/GlobalSearchBar.res

```diff
@@ -178,15 +185,38 @@ let make = () => {
     setSearchText(_ => value)
   }, ~wait=500)
 
+  let onFilterClicked = category => {
+    let newFilter = category.categoryType->getcategoryFromVariant
+    let lastString = searchText->String.charAt(searchText->String.length - 1)
+    if activeFilter->isNonEmptyString && lastString !== ":" {
```

Comment:
please define ":" as constant string
let semiColon=":"
similarly for all the string matchs